Baby Monitor/Source Files/Modules/Server/ServerViewController.swift

```diff
@@ -45,6 +45,15 @@ final class ServerViewController: BaseViewController {
         style: .plain,
         target: nil,
         action: nil)
+
+    private lazy var debugInfoLabel: UILabel = {
```

Comment:
I wonder if this could be simplified into a dynamic var:

```
var debugInfoLabel: UILabel {
        let label = UILabel()
        label.numberOfLines = 0
        label.textColor = .white
        label.textAlignment = .center
        return label
}
```

Reply:
Yes, but at the same time there is convention of lazy var's in the project from what I can see, so I thought to keep the convention.

Reply:
Sure, let's leave it. In both cases (lazy or not) the label will be re-created each time the var is called. The proposed solution is just tiniest bit simpler ;)